jitpack.yml

```diff
@@ -0,0 +1,6 @@
+jdk:
```

Comment:
Ahh, so this is a trick to tell jitpack how to find android sdk in repo 😮

Reply:
🪄